sceptre/cli/diff.py

```diff
@@ -133,13 +135,15 @@ def diff_command(
 def output_diffs(
     diffs: Iterable[StackDiff],
     writer_class: Type[DiffWriter],
+    no_colour: bool,
     output_buffer: TextIO,
     output_format: str,
 ) -> int:
     """Outputs the diff results to the output_buffer.
 
     :param diffs: The differences computed
     :param writer_class: The DiffWriter class to be instantiated for each StackDiff
+    :param no_colour: The no_colour attribute from the Sceptre context
```

Comment:
I think this docstring should indicate what it _does_ rather than where it's from.